test/integration/indexer/links_lookup_test.rb

```diff
@@ -120,6 +125,9 @@ def test_extracts_parts_of_taxonomy
       "content_id" => grandparent_1_content_id,
       "base_path" => "/grandparent-1",
       "title" => "Grandparent 1",
+      "details" => {
```

Comment:
In the meantime I'm going to raise a separate PR to make the gem not assume `internal_name` already exists, as it's supposed to optional according to the schema.